### ISSUE TITLE
Syntactically invalid export to RDF

### ISSUE THREAD

**Issue by LaurensRietveld** [rdfhdt/hdt-cpp#85]
The serdi serializer does not always return syntactically valid RDF. Luckily we reverted the latest release so we can still fix this ;)

```
 echo '<http://sub> <http://pred> "@some-string" .' > test.nt && rdf2hdt test.nt test.hdt && hdt2rdf test.hdt - | serdi  -
<http://sub> <http://pred> "@some-string\""@some-string .
error: (stdin):1:56: missing ';' or '.'
```

**Comment by drobilla** [comment]
This exact same problem seems to happen with the raptor parser/serializer as well.  The serd one chops up string literals in the same way.

**Comment by drobilla** [comment]
#86

**Comment by RubenVerborgh** [comment]
Thanks! @LaurensRietveld, can you verify #86?

**Comment by LaurensRietveld** [comment]
Look good to me. As @drobilla mentioned, the raptor serializer fails as well. And, as raptor takes precedence over raptor when serializing (see https://github.com/rdfhdt/hdt-cpp/blob/master/hdt-lib/src/rdf/RDFSerializer.cpp#L18), the serialization might still produce invalid output when raptor is listed as dep. I propose to tackle this issue in #77, by removing raptor completely for serialization.

**Comment by RubenVerborgh** [comment]
Okay, will go ahead and merge #86 then (and fix #77 soon after).
